db/migrate/20180329205900_rename_comprehension_explanation_to_information.rb

```diff
@@ -0,0 +1,20 @@
+class RenameComprehensionExplanationToInformation < ActiveRecord::Migration[5.1]
+  def up
+    rename_column :course_assessment_question_text_response_compre_solutions,
+                  :explanation,
+                  :information
+    change_column :course_assessment_question_text_response_compre_solutions,
+                  :information,
+                  :string,
+                  limit: 255
```

Comment:
Can you put down why you changed `information`'s type to string? Will the migration fail because of existing entries that are longer than 255? Should your migration deal with those cases?

Reply:
Changed to string as there is no need for summernote.
I set the length to 255 as I went to run `\d course_assessment_question_text_response_compre_solutions` and noticed that the type for `solution` was `character varying(255)[]`, so I wanted to make both `solution` and `information` similar.

Reply:
I have removed the length `limit`.